site/docs/commands/test.md

```diff
@@ -31,6 +31,14 @@ very_good test [arguments]
 Run "very_good help" to see global options.
 ```
 
+### Passing Flutter specific arguments
+
+The `flutter test` command exposes more arguments than those available through `very_good test`. Despite this, you can use the argument terminator `--` to signify the end of `very_good test` command options and the beginning of `flutter test` command options.
+
+For example, if you wish to run `flutter test --no-track-widget-creation` you can simply do `very_good test -- --no-track-widget-creation`.
+
+You may also target specific files by using the argument terminator; for example `very_good test -- test/very_good_test/**`. However, when targeting a subset of the project tests optimization will always be disabled.
```

Comment:
Small grammatical changes here (commas, and changed "you can simply do" to "you can simply use" here's the revised version:

For example, if you wish to run flutter test --no-track-widget-creation, you can simply use very_good test -- --no-track-widget-creation.

You may also target specific files by using the argument terminator; for example, very_good test -- test/very_good_test/**. However, when targeting a subset of the project's tests, optimization will always be disabled.